README.md

```diff
@@ -36,6 +36,11 @@ theme = "pixyll"
   twitter_username = "username"
   disqus_shortname = "sitename"
   paginate = true
+
+[[menu.main]]
```

Comment:
please add this menu to exampleSite configuration as well to show the support for menu items